### DISCUSSION TITLE
What's the command to install the torch packages on Windows?

### DISCUSSION THREAD

**Discussion by scoroman** [yl4579/StyleTTS2#84] [Q&A]
Hi, pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118 -U doesn't work. i was wondering what the cmd is? I put in pip install torch with the index for the torch url but that didnt work either

**Comment by Kreevoz** [comment]
That should work, provided you got python installed on your machine and added to the Path environment variable.
If you type `python` into a command window of your choice and don't get a python prompt, it isn't installed properly.

**Comment by scoroman** [comment]
Unfortunately it doesn't work. I have my python installed with pip and the
interpreter working fine. see attached

**Reply by Kreevoz** [reply]
Don't know what your problem is specifically, but you can also set up a conda environment and grab python + dependencies that way. Have a look around on youtube, there are literally _tons_ of tutorials for that.

**Reply by rikabi89** [reply]
I had a ton of issue with getting the right cuda version to work with this package. The one on the readme kept reverting to CPU only. Anyway, this package worked for me on anaconda : torch==1.13.1, torchaudio==0.13.1, torchvision==0.14.1
